README.md

```diff
@@ -172,10 +172,34 @@ it("upload assets", () => {
 
 ## Development
 
+### Setup
+
+1. Install dependencies: `npm install`
```

Comment:
I tink it would be better to use `npm ci` instead - we should expect developers/customers to install dependencies declared on the package-lock.json. npm install installs also package updates